docs/how-to/evaluate/custom_evaluation_pipeline.md

```diff
@@ -0,0 +1,8 @@
+# How to create custom Evaluation Pipeline for Ragbits evaluation
+
+Ragbits provides a ready-to-use evaluation pipeline for document search, implemented within the `ragbits.evaluate.document_search.DocumentSearchPipeline` module.
+
+To create a custom evaluation pipeline for your specific use case, you need to implement the `__call__` method as part of the `ragbits.evaluate.pipelines.base.EvaluationPipeline` interface.
+
+
+Please find the [working example](optimize.md#define-the-optimized-pipeline-structure) here
```

Comment:
I don't think linking to the optimizer is enough here, we should have a standalone example going in-detail about pipelines.

Reply:
Resolved in https://github.com/deepsense-ai/ragbits/pull/217/